File: tournaments/ratingThreshold/ratingThreshold.js
function ratingThreshold(threshold, ratings) {
    const result = [];
    
    for (let i = 0; ratings.length > i; ++i) {
        const item = ratings[i];
        const s = item.reduce((a, b) => a + b, 0);
        
        if (threshold > s / item.length) {
            result.push(i);
        }
    }
    
    return result;
}

File: tournaments/ratingThreshold/ratingThreshold.cs
int[] ratingThreshold(double threshold, int[][] ratings) {
    List<int> result = new List<int>();

    for (int i = 0; ratings.Length > i; ++i) {
        int[] item = ratings[i];
        int s = item.Sum();

        if (threshold > ((Double)s / item.Length)) {
            result.Add(i);
        }
    }

    return result.ToArray();
}
